tutorials/apply_force_torque.md

```diff
@@ -0,0 +1,115 @@
+\page apply_force_torque Apply Force and Torque
```

Comment:
I think you'll need to add a link to this page in https://github.com/gazebosim/gz-sim/blob/gz-sim7/tutorials.md.in